Signal/translations/en.lproj/Localizable.strings

```diff
@@ -1540,6 +1540,9 @@
 /* action sheet item */
 "SHARE_ACTION_TWEET" = "Twitter";
 
+/* alert body when sharing filefailed because of untrusted/changed identity keys */
+"SHARE_EXTENSION_FAILED_SENDING_BECAUSE_UNTRUSTED_IDENTITY_KEY" = "Your safety number with %@ has recently changed. You may wish to verify it in the main app before resending.";
```

Comment:
TIOLI - I tend to use a `_FORMAT` suffix for l10n strings that should be formatted.

Reply:
Done!